### ISSUE TITLE
Use custom browser defined in JOSM

### ISSUE THREAD

**Issue by iriman** [JOSM/Mapillary#96]
In JOSM we can define custom browser ([ticket](https://josm.openstreetmap.de/ticket/16891)) but when I click on _View in browser_ it opens default Windows browser.

![image](https://user-images.githubusercontent.com/8467903/64111993-456e6b00-cd9b-11e9-913b-cdbdcaf6d8f7.png)

**Comment by miurahr** [comment]
There is a browser function in utils class.
https://github.com/JOSM/Mapillary/blob/9f4f231fbf816e3d160ebed3ce1832eaa30ede9d/src/main/java/org/openstreetmap/josm/plugins/mapillary/utils/MapillaryUtils.java#L46
And its core logic is as follows;
```
 if (desktop.isSupported(Desktop.Action.BROWSE)) {
        desktop.browse(url.toURI());
 }
```

We can use org.openstreetmap.josm.tools.OpenBrowser class to enable custom browser.
It may be easy and good for new contributors first patch.

**Comment by GhostFoxSledgehammer** [comment]
This was fixed in 9dc0ac9
